### ISSUE TITLE
Question: Mono and IKVM.Reflection

### ISSUE THREAD

**Issue by borgdylan** [jfrijters/Managed.Reflection#3]
Is this project a netstandard port of IKVM.Reflection/Emit? Does it keep mono in mind i.e using managed crypto and not CNG. I have a compiler that uses IKVM.Reflection to output assemblies and I was looking to have portable PDB output. I also wanted to be able to port it to .NET Core and IKVM.Reflection was the only roadblock to doing so. If this project is similar and compatible in API structure I will not hesitate to switch to it for IL and pdb generation.

**Comment by jfrijters** [comment]
This is indeed the successor to IKVM.Reflection that targets .NET Standard. IKVM.Reflection will continue to exist for a while (as it targets .NET 2.0). I have not tried Mono and I did remove a few (old) Mono specific workarounds. If you try it and have issues on Mono, I'd be happy to address them.
The API structure is the same as IKVM.Reflection. The major difference is that the _IKVM_ namespace was changed to _Managed_. A few APIs were removed:
- deprecated APIs
- Module.GetSignerCertificate()
- CAS permission related stuff
- IResourceWriter based DefineResource() APIs
